plugins/providers/bigquery/provider.go

```diff
@@ -143,10 +143,11 @@ func (p *Provider) GetResources(pc *domain.ProviderConfig) ([]*domain.Resource,
 			dataset := d.ToDomain()
 			dataset.ProviderType = pc.Type
 			dataset.ProviderURN = pc.URN
-			var fetchTables bool
+			fetchTables := true
```

Comment:
Can you add some comments in this function regarding its flow.

Reply:
I have added some comments